### ISSUE TITLE
Are there certain words that cannot be used in the name of a file or folder ?

### ISSUE THREAD

**Issue by adioni** [typemill/typemill#262]
Testing out typemill and I've noticed if I  try and create a new file / folder within the content GUI page, certain words get removed in the process.

For example if the name a new page "what-is-a-man" the resultant page url becomes http://example.com/what-man

Same thing if I create a folder called "in the beginning"; the resultant url becomes just http://example.com/beginning

Not sure if I overlooked something in the docs, but is this the expected behavior? Is there a way to override it if I explicitly want those helper words in the url ?

**Comment by trendschau** [comment]
you are right, it is done by the library urlify, I will check that for release 1.4.7 (the release 1.4.6 is already closed) and look if there are options for stop-words...

**Comment by trendschau** [comment]
I cannot change this behavior easily because it is done by an external library. But in version 1.4.9 you can rename the files and folders in the meta-tab. That should be a workaround. I will check if the library has some options to disable stop-words.
